components/Comment/lib/CommentModel.ts

```diff
@@ -0,0 +1,100 @@
+import { formatDateStandard, timeSince } from "~/config/utils/dates";
```

Comment:
I'm not a big fan of having classes just to format data from backend to serve FE purposes. 
I doesn't really do much more than a simple object, and classes are harder to maintain